### DISCUSSION TITLE
Less gene expression in multiomics feature plot

### DISCUSSION THREAD

**Discussion by Abigail575** [stuart-lab/signac#1486] [Q&A] [answered]
Hi,
When I look for marker gene expression in my multimodal feature plots, I observe much less expression than when I look at the same genes on scRNA feature plots of the same data. Do you know why this is? (I assume the differences observed due to the different quality control steps in the vignettes would not solely account for this.)
Thanks!
Abigail

**Comment by timoast** [comment, marked as answer]
Multiome is single nucleus, scRNA-seq is single cell. You tend to get better RNA data from whole cells rather than nuclei. The multiome assay is also more difficult as it needs to be optimized for the ATAC fragment recovery, not just RNA, so may not be as highly sensitive as snRNA-seq either.
